docs/pages/get-started/installation.md

```diff
@@ -40,7 +40,7 @@ Verify that the installation was successful by running `expo whoami`. You're not
 The fastest way to get up and running is to use the Expo Go app on your iOS or Android device. Expo Go allows you to open up apps that are being served through Expo CLI.
 
 - 🤖 [Android Play Store](https://play.google.com/store/apps/details?id=host.exp.exponent) - Android Lollipop (5) and greater.
-- 🍎 [iOS App Store](https://itunes.com/apps/exponent) - iOS 11 and greater.
+- 🍎 [iOS App Store](https://apps.apple.com/jp/app/expo-go/id982107779) - iOS 11 and greater.
```

Comment:
```suggestion
- 🍎 [iOS App Store](https://apps.apple.com/app/expo-go/id982107779) - iOS 11 and greater.
```
I'd remove the locale from the URL because otherwise it shows Japanese version to everyone 😅

Reply:
Oops